src/prepack-standalone.js

```diff
@@ -13,22 +13,12 @@ import construct_realm from "./construct_realm.js";
 import initializeGlobals from "./globals.js";
 import * as t from "babel-types";
 import { getRealmOptions, getSerializerOptions } from "./options";
-import { type ErrorHandler } from "./errors.js";
+import { type ErrorHandler, fatalError } from "./errors.js";
 
 import type { Options } from "./options";
 import { defaultOptions } from "./options";
 import type { BabelNodeFile, BabelNodeProgram } from "babel-types";
 
-// This should just be a class but Babel classes doesn't work with
-// built-in super classes.
-export function InitializationError() {
```

Comment:
This is a breaking change to Prepack users, include our website:
https://github.com/facebook/prepack/blob/gh-pages/js/repl-worker.js#L31

Landing this change needs to get synchronized with updating the prepack version on the website, and the repl-worker.js code.